config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,6 @@
+# Be sure to restart your server when you modify this file.
+
+# ApplicationController.renderer.defaults.merge!(
+#   http_host: 'example.org',
+#   https: false
+# )
```

Comment:
We've tended to keep unused configuration files around, to reduce noise when upgrading.